cc/crypto/hpke/recipient_context.h

```diff
@@ -32,6 +32,7 @@ struct KeyPair {
   std::string private_key;
   std::string public_key;
 
+  // Generates a KeyPair that the recipient and sender can use for HPKE.
```

Comment:
This keypair will only be used by the recipient, since sender's key will use ephemeral keys generated during `setup_base_sender`

Reply:
Fixed.